djangocms_versioning/templates/admin/djangocms_versioning/change_list.html

```diff
@@ -1,2 +1,18 @@
 {% extends "admin/change_list.html" %}
```

Comment:
I don't think that we need any of the changes in this file. This is for the page tree.